From: Jeremy Bicha <jbicha@debian.org>
Date: Thu, 10 Mar 2022 08:17:11 -0500
Subject: Revert "status/bluetooth: Show bluetooth-disabled icon when
 disabled"

This reverts commit 8a7f93596992000942cdc1337f45ee3197376351.

buntu 22.04 LTS isn't using the new gnome-bluetooth series
---
 js/ui/status/bluetooth.js | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/js/ui/status/bluetooth.js b/js/ui/status/bluetooth.js
index 714e3ea..98ccc3d 100644
--- a/js/ui/status/bluetooth.js
+++ b/js/ui/status/bluetooth.js
@@ -38,6 +38,7 @@ class Indicator extends PanelMenu.SystemIndicator {
         this._proxy.connect('g-properties-changed', this._queueSync.bind(this));
 
         this._item = new PopupMenu.PopupSubMenuMenuItem(_("Bluetooth"), true);
+        this._item.icon.icon_name = 'bluetooth-active-symbolic';
 
         this._toggleItem = new PopupMenu.PopupMenuItem('');
         this._toggleItem.connect('activate', () => {
@@ -142,9 +143,6 @@ class Indicator extends PanelMenu.SystemIndicator {
         else
             this._item.visible = this._proxy.BluetoothHasAirplaneMode && !this._proxy.BluetoothAirplaneMode;
 
-        this._item.icon.icon_name = adapterPowered
-            ? 'bluetooth-active-symbolic' : 'bluetooth-disabled-symbolic';
-
         if (nConnectedDevices > 1)
             /* Translators: this is the number of connected bluetooth devices */
             this._item.label.text = ngettext('%d Connected', '%d Connected', nConnectedDevices).format(nConnectedDevices);
